coopr-docs/docs/source/guide/superadmin/chef-solo-automator-plugin.rst

```diff
@@ -227,16 +295,18 @@ The example coopr service definition invoking this cookbook is called "base". It
 action of type "chef-solo" and run_list ``recipe[coopr_base::default]`` (which includes ``recipe[coopr_hosts::default]``).
 Note that the community "hostsfile" cookbook is not needed in the run-list since it is declared in coopr_hosts's metadata.
```

Comment:
Chef docs use 'run-list' when referring to it in the general sense, which i tried to be consistent with.  But our json field is "run_list".  So something may still be incorrect but i'm not sure